src/web3-store.js

```diff
@@ -95,7 +95,7 @@ export const createStore = () => {
   return {
     setBrowserProvider,
     setProvider,
-    close,
+    disconnect,
```

Comment:
I think we should leave close as an alias not to break code on upgrade so brutally. close can console.log deprecated and call disconnect

Reply:
I think we don't need the log then, unless you plan to change the function name to **disconnect** in the future.